### DISCUSSION TITLE
posts/webtop-container/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#222] [Comments]
# posts/webtop-container/

Documentation for Techno Tim YouTube videos for all your copy pasta needs!

https://technotim.live/posts/webtop-container/

**Comment by rventura520** [comment]
How i coud deploy this with a macvlan network?

**Comment by tgechev** [comment]
Hi Tim, I recently discovered your YouTube channel and blog. I really enjoy the content and try out most of the things you show.
Right now I am stuck with a problem I suspect is related to the webtop setup mentioned in this video. While I have successfully deployed the webtop container and logged into Ubuntu, I am not able to run .AppImage applications. I have used the linuxserver/ubuntu-kbe image which comes with Ubuntu 22.04. I have read online that .AppImage programs require FUSE and that version 22.04 might not come with it. I have tried all the solutions I managed to find for this including installing only the libfuse2 package and not the actual fuse package. However, nothing seems to work. When I try to run the .AppImage program from the terminal I get:
*fuse: device not found, try 'modprobe fuse' first*
then for 'modprobe fuse' I get:
*modprobe: FATAL: Module fuse not found in directory /lib/modules/5.13.x*
then I found about depmod and ran 'sudo depmod' which outputs:
*depmod: ERROR: could not open directory /lib/modules/5.13.x: No such file or directory
depmod: FATAL: could not search modules: No such file or directory*

It turns out the entire lib/modules folder is missing and most likely this is why fuse is not working properly. Then at this point I remembered your video about netboot.xyz which fortunately I have already set up. I booted the Ubuntu-kbe 22.04 liveCD just to check if the lib/modules folder exists and in fact it was there. I did not have to install FUSE manually since it was also present. I downloaded the same .AppImage program I wanted to use from the webtop Ubuntu, marked it as executable and it launched right up. All this makes me think the webtop image from linuxserver is somehow set up differently. Since I couldn't find a solution for this I decided to write here hoping someone might have already faced the same problem and solved it. I need to be able to run .AppImage programs from the webtop Ubuntu since I am running the webtop on a NAS device and need to access files through the .AppImage program which are stored on the NAS.

Any help is very much appreciated,
Cheers!
